eng/release.yml

```diff
@@ -0,0 +1,200 @@
+trigger: none
+pr: none
+
+parameters:
+- name: IsTestRun
+  type: boolean
+  default: true
+- name: IsDryRun
+  type: boolean
+  default: true
+
+variables:
+- name: _TeamName
+  value: DotNetCore
+  readonly: true
+- group: Release-Pipeline
+- name: IsDryRun
+  value: ${{ parameters.IsDryRun }}
+  readonly: true
+- name: IsTestRun
+  value: ${{ parameters.IsTestRun }}
+  readonly: true
+
+resources:
+  pipelines:
+  - pipeline: Build
+    source: dotnet-dotnet-monitor
+
+stages:
+- stage: Validation
+
+  pool:
+    name: NetCore1ESPool-Internal
+    demands: ImageOverride -equals Build.Windows.10.Amd64.VS2019
+
+  jobs:
+  - job: Validate
+
+    variables:
+    # Allow for differentiation of runs of this pipeline
+    # when running it with the same build repeatedly.
+    - name: RunRevision
+      value: $[counter(format('{0}|{1}|{2}', variables['resources.pipeline.Build.runID'], variables['IsDryRun'], variables['IsTestRun']), 1)]
+      readonly: true
+
+    workspace:
+      clean: all
+
+    steps:
+    - download: none
+
+    - task: PowerShell@2
+      displayName: Get BAR ID
+      inputs:
+        filePath: $(Build.SourcesDirectory)/eng/release/Scripts/GetBarId.ps1
+        arguments: >-
+          -BuildId $(resources.pipeline.Build.runID)
+          -TaskVariableName 'BarId'
+      env:
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+
+    - task: PowerShell@2
+      displayName: Get Release Version
+      inputs:
+        filePath: $(Build.SourcesDirectory)/eng/release/Scripts/GetReleaseVersion.ps1
+        arguments: >-
+          -BarId $(BarId)
+          -MaestroToken $(MaestroAccessToken)
+          -TaskVariableName 'ReleaseVersion'
+    
+    - task: PowerShell@2
+      displayName: Get Build Version
+      inputs:
+        filePath: $(Build.SourcesDirectory)/eng/release/Scripts/GetBuildVersion.ps1
+        arguments: >-
+          -BarId $(BarId)
+          -MaestroToken $(MaestroAccessToken)
+          -TaskVariableName 'BuildVersion'
+
+    - powershell: |
+        $buildName = "${env:ReleaseVersion} [${env:BuildVersion}]"
+        if ($env:IsDryRun -eq 'true') {
+          $buildName += "[Dry]"
+        }
+        if ($env:IsTestRun -eq 'true') {
+          $buildName += "[Test]"
+        }
+        $buildName += "[Run ${env:RunRevision}]"
+        Write-Host "##vso[build.updatebuildnumber]$buildName"
+      displayName: Set Name
+
+- stage: Publish
+  dependsOn:
+  - Validation
+
+  pool:
+    name: NetCore1ESPool-Internal
+    demands: ImageOverride -equals Build.Windows.10.Amd64.VS2019
+
+  jobs:
+  - deployment: PublishToStorageAccounts
+    displayName: Publish to Storage Accounts
+
+    ${{ if eq(parameters.IsTestRun, 'true') }}:
+      environment: Diagnostics-Monitor-Storage-Test
+    ${{ else }}:
+      environment: Diagnostics-Monitor-Storage-DotNetCli
+
+    variables:
+    - ${{ if eq(parameters.IsTestRun, 'true') }}:
+      - group: DotNet-Diagnostics-Storage-Test
+    - group: DotNet-DotNetStage-Storage
```

Comment:
This is meant to be additive right?

Reply:
Correct. It will always require this group since it will always copy from dotnetstage account.